### DISCUSSION TITLE
Instructions for CNAME file under File Manager in the root files

### DISCUSSION THREAD

**Discussion by spsrinivas** [GetPublii/Publii#1335] [General] [answered]
I've moved away from Github to Git due to deprecation plans. 

Every time I Sync, the custom domain is overwritten with GitHub Pages domain without the CNAME file. What exactly needs to be there in the file and what should be the file name ? would this work? 

File name: CNAME
Content: CNAME example.com

**Comment by dziudek** [comment, marked as answer]
Hi,

You must put this file under Tools -> File Manager -> root files

**Reply by example-user30** [reply]
Will Publii support the management of the CNAME file in the future?  Thanks

**Reply by dziudek** [reply]
@example-user30 - you can manage this file under File Manager, so there is no need for additional UI for this. The same for robots.txt, humans.txt, security.txt, _redirects etc.

**Comment by spsrinivas** [comment]
Thanks. Would the file name and format as mentioned above work?

**Reply by dziudek** [reply]
If it is the same as file created by Github during custom domain configuration, then it should work
